bin/all_sky_search/pycbc_plot_kde_vals

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import h5py, argparse
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file')
+parser.add_argument('--template-file', required=True)
+parser.add_argument('--param', nargs='+', required=True, help='mchirp, eta, chi_eff')
```

Comment:
I think the code should be able to handle the case of parameters with different names, as long as the input files have datasets with these names.  The help should say that you can specify one parameter to plot vs the kde, if the plot type is kde_vs_param, or two parameters separated by a space if the plot type is param_vs_param.  If the user specifies a third parameter it looks like the code doesn't do anything with it ...